docs/root/configuration/runtime.rst

```diff
@@ -19,7 +19,24 @@ Virtual file system
 Layering
 ++++++++
 
-The runtime can be viewed as virtual file system consisting of multiple layers:
+The runtime can be viewed as a virtual file system consisting of multiple layers. The :ref:`layered
+runtime <envoy_api_msg_config.bootstrap.v2.LayeredRuntime>` bootstrap configuration specifies this
+layering. Runtime settings in later layers override earlier layers. A typical configuration might
```

Comment:
Isn't this inconsistent with the rest of our static config?
I was a bit surprised in the LDS PR that one couldn't override bootstrap listeners with further dynamic updates (just not how we did things in-house) and while I'd still prefer we always allow dynamic config to update command line config, I also think it'd be confusing it we allow it some places and not others

Reply:
This is a bit different, in that it will be the operator that explicitly states whether override is possible. They will control the layering. I agree that in theory, it might have been nice to structure all the xDS APIs as overlay layers with explicit control, but that's a fairly disruptive change with no requestor today. For TDS and runtime, Lyft (Matt basically) have asked for this level of control over how layering works.

Reply:
Thinking about this further, I think the real distinguishing feature here is that runtime is actually much more operationally complicated than other xDS, specifically, we have at least 4 different sources (admin, disk, static, dynamic xDS),  This creates a situation where it is much more likely that an operator will not be satisfied by a fixed overlay order.

Reply:
I guess my thought is if we're going to treat it differently maybe we could name it differently?  inline config or something?  I'm 100% Ok with the behavior, just find the behavior-and-naming inconsistent

Reply:
Here are options to use for naming this layer
1. `static`
2. `inline`
3. `base`

which do you folks prefer @alyssawilk @mattklein123? I'm good with whatever makes the most people the most happy, this is all potatoes potatahs to me.

Reply:
I don't have a strong preference for the name of this layer. I guess either static or base is preferred over inline.

The historical context on why we don't allow LDS and CDS updates to update bootstrap is config is that it's potentially confusing if an xDS server does not return a static resource, thus causing it to get deleted. Also, some things won't be easy to update without a tremendous amount of thinking (e.g., the xDS cluster itself, stats, etc.). It would not be difficult at all to allow static LDS/CDS to get updated by the API, but IMO it would need to be opt-in on a per-cluster basis, but that would be trivial to implement. @alyssawilk if you want that I would open a separate issue and we can discuss?

Reply:
I left this as `static_layer` and a clarifying comment.